schema/models_directory/pb/qm/dft/ksdft/double_hybrid/tags.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "allOf": [
+        {
+            "$ref": "../../../../../../model/tags.json"
+        }
+    ],
+    "description": "Tags specific to hybrid functionals only.",
+    "items": {
+        "anyOf": [
+            {
+                "$ref": "definitions.json#/tags"
+            },
+            {
+                "description": "QM definitions",
+                "$ref": "../../../definitions.json#/scaling/3"
```

Comment:
Is the scaling level `3` correct here and elsewhere in `ksdft` folder?

Reply:
For most of them I would say so, yes (see also [here](http://iopenshell.usc.edu/chem545/material/overviewDFTSousa.pdf) ). I would assume that double hybrid functionals show a higher scaling due to evaluation of MP2 energy terms (although I didn't find a corresponding reference in my quick google search).